content/applications/inventory_and_mrp/inventory/management/incoming/one_step.rst

```diff
@@ -43,7 +43,11 @@ on the dashboard, you can click the *1 TO PROCESS* button under the
 
 Now, enter the picking that you want to process. You will be able to
 click on *Validate* to complete the move directly as products coming
-from suppliers are considered as being always available.
+from suppliers are considered as being always available. 
+
+Note: if you have Storage Locations activated, you would click the 
+hamburger menu next to the Done quantity to specify the location(s) 
+where you are storing the received product(s).
```

Comment:
- [RST cheat sheet: notes admonitions](https://www.odoo.com/documentation/13.0/contributing/documentation/rst_cheat_sheet.html#note)
- RST guidelines: break the line at the 100th character
- not documented yet but we'll use `guilabel` for elements that are part of the interface (fields names, buttons, views etc)

Does this suggestion seem good to you?
```suggestion
.. note::
   If you have :guilabel:`Storage Locations` activated, you can click the hamburger menu next to 
   the :guilabel:`Done quantity` to specify the location(s) where you are storing the received
   product(s).
```